audio/vcvrack/Portfile

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        VCVRack Rack 1.1.6 v
+name                vcvrack
+
+categories          audio
+platforms           darwin
+license             GPL-3
+
+set rack_fundamental_version    1.3.1
+
+homepage            https://vcvrack.com/
+
+description         VCV Rack is an open-source virtual modular synthesizer.
+
+long_description    {*}${description}
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+# Build process requires git checkout
+fetch.type          git
+
+patchfiles          patch-makefile.diff
+
+depends_build-append \
+                    port:autoconf \
+                    port:automake \
+                    port:cmake \
+                    port:jq \
+                    port:python38 \
```

Comment:
Does it use autoconf, automake, python38? I didn't see it in the log, but it is downloading and building lots of other dependencies and is using silent rules so the exact commands it's running aren't always visible.

Reply:
You're right, I removed those and the build still completed.  Will see how it fares in CI.

Reply:
And jq? I see it is mentioned in the [building instructions](https://vcvrack.com/manual/Building). I don't see it used in the log but maybe it is still used in some way that isn't shown in the log.

Reply:
`jq` is indeed used by one of the dependencies that gets pulled in, "jpommier".